docs/resources/Guild.md

```diff
@@ -376,7 +376,85 @@ Create a new guild. Returns a [guild](#DOCS_RESOURCES_GUILD/guild-object) object
 
 ## Get Guild % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}
 
-Returns the [guild](#DOCS_RESOURCES_GUILD/guild-object) object for the given id.
+Returns the [guild](#DOCS_RESOURCES_GUILD/guild-object) object for the given id. If `with_counts` is set to `true`, this endpoint will also return `approximate_member_count` and `approximate_presence_count` for the guild.
+
+###### Query String Params
+
+| Field       | Type    | Description                                                                   | Required | Default |
+|-------------|---------|-------------------------------------------------------------------------------|----------|---------|
+| with_counts? | boolean | when `true`, will return approximate member and presence counts for the guild | false    | false   |
+
+###### Example Response
+
+```json
+{
+    "id": "2909267986263572999",
+    "name": "Mason's Test Server",
+    "icon": "389030ec9db118cb5b85a732333b7c98",
+    "description": null,
+    "splash": "75610b05a0dd09ec2c3c7df9f6975ea0",
+    "discovery_splash": null,
+    "approximate_member_count": 2,
+    "approximate_presence_count": 2,
+    "features": [
+        "INVITE_SPLASH",
+        "VANITY_URL",
+        "COMMERCE",
+        "BANNER",
+        "NEWS",
+        "VERIFIED",
+        "VIP_REGIONS"
+    ],
+    "emojis": [
+        {
+            "name": "ultrafastparrot",
+            "roles": [],
+            "id": "393564762228785161",
+            "require_colons": true,
+            "managed": false,
+            "animated": true,
+            "available": true
+        }
+    ],
+    "banner": "5c3cb8d1bc159937fffe7e641ec96ca7",
+    "owner_id": "53908232506183680",
+    "application_id": null,
+    "region": "us-east",
+    "afk_channel_id": null,
+    "afk_timeout": 300,
+    "system_channel_id": null,
+    "widget_enabled": true,
+    "widget_channel_id": "639513352485470208",
+    "verification_level": 0,
+    "roles": [
+        {
+            "id": "290926798626357250",
```

Comment:
isn't @everyone role id = guild id?
```suggestion
            "id": "2909267986263572999",
```

Reply:
Oh I faked the ID which is why it's different.